test/cpp/api/modules.cpp

```diff
@@ -1116,6 +1116,18 @@ TEST_F(ModulesTest, LogSigmoid) {
   ASSERT_TRUE(torch::allclose(y, y_exp, 1e-4, 1e-7));
 }
 
+TEST_F(ModulesTest, Softmax) {
+  Softmax m(SoftmaxOptions(1));
```

Comment:
We can change this to
```suggestion
  Softmax m(/*dim=*/1);
```
to also test `SoftmaxOptions`'s implicit constructor. ;)

Reply:
This is also fixed as well.